index.js

```diff
@@ -53,71 +208,104 @@ module.exports = {
     // Requiring true to be set as the value will help future-proof our code.
     'react/jsx-boolean-value': ['error', 'always'],
 
-    // don't require trailing commas for multi-line function calls
-    // airbnb config assumes you're transpiling with https://npmjs.com/babel-preset-airbnb
-    // which includes trailing function commas.
-    // https://github.com/eslint/eslint/issues/7571#issuecomment-259538272
-    // This makes life harder if you're linting bin files that aren't transpiled
-    // or have the trailing function (ES2017 proposal) config set.
-    // This is bad as that adds friction to linting all files, which is desired.
-    'comma-dangle': ['error', {
-      arrays: 'always-multiline',
-      objects: 'always-multiline',
-      imports: 'always-multiline',
-      exports: 'always-multiline',
-      functions: 'never',
-    }],
+    // React fragment syntax requires Babel 7.x but this preset needs to still support Babel 6.x
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md
+    'react/jsx-fragments': 'off',
 
-    // airbnb defaults to a maximum cyclomatic complexity of 11
-    complexity: ['error'],
+    // It is common practice to use .js for React
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md
+    'react/jsx-filename-extension': 'off',
+
+    // Disabling this rule until this is resolved https://github.com/yannickcr/eslint-plugin-react/issues/1848
+    // at the moment the fix makes the code look messy and at times unreadable
+    'react/jsx-one-expression-per-line': 'off',
+
+    // AirBnB uses 'error' but it's too disruptive during development
+    // and we shouldn't prevent compilation because of it
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unused-prop-types.md
+    'react/no-unused-prop-types': ['warn'],
+
+    // THIS IS A CONTROVERSIAL RULE THAT WE SHOULD DISCUSS
```

Comment:
PR review notes should be omitted.

Reply:
https://github.com/americanexpress/eslint-config-amex/pull/28